conf/joystick/xbox_gamepad.xml

```diff
@@ -84,7 +84,7 @@ so e.g. HatDown(dpad)
 
     <!-- go to "land here" block if in AUTO2 and pressing down on dpad -->
     <message class="ground" name="JUMP_TO_BLOCK" on_event="(mode > 1) && HatDown(dpad)">
-      <field name="block_id" value="IndexOfBlock('land here')"/>
+      <field name="block_id" value="IndexOfBlock('Land here')"/>
```

Comment:
This will make this unusable for all other flight plans except yours. I don't mind changing the name of the land block, but then it has to be done everywhere.

Reply:
I changed this back to the lowercase 'land here' (also in the flight plan) to keep things compatible with other flight plans that may not have been merged into master yet.